.github/workflows/check_generated_code_drift.yml

```diff
@@ -30,8 +30,14 @@ jobs:
         id: changed-files
         uses: tj-actions/changed-files@v25
         with:
+<<<<<<< HEAD
```

Comment:
You might need to resolve this merge conflict first

Reply:
Ah thanks, I missed that one